tests/foreman/maintain/test_upgrade.py

```diff
@@ -171,27 +171,30 @@ def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
     )
 
 
-@pytest.mark.stubbed
 @pytest.mark.include_capsule
-def test_positive_self_update_for_zstream(sat_maintain):
-    """Test satellite-maintain self-upgrade to update maintain packages from zstream repo.
+def test_positive_self_update_maintain_package(sat_maintain):
```

Comment:
so there's confusion in f-m package updates, major version package upgrade is called self-upgrade, so to avoid confusion, can we add for zstream back? 
```suggestion
def test_positive_self_update_maintain_package_for_zstream(sat_maintain):
```

Reply:
well technically the auto update could run z-stream or y-stream upgrade for foreman-maintain. The self-upgrade command only applies to y-stream, while auto-update can do any of the upgrades.

Reply:
well, they are not same, for y-stream self-upgrade enables next version repository and use `dnf --enablerepo..` to update the maintain packages, and for auto-update its just updates maintain packages if its available in the enabled repo. Let me know wdyt, I don't want to block this PR just for this.